stack/platforms.rst

```diff
@@ -5,27 +5,43 @@ Supported Platforms
 Introduction
 ============
 
-The LSST Data Management software is required to be portable across many compute platforms ranging from high-performance computing centers to laptops (DMS-REQ-0308).
+The LSST Data Management software is required to be portable across many compute platforms ranging from high-performance computing centers to laptops (refer to DMS-REQ-0308 in :lse:`61`).
```

Comment:
```suggestion
The Science Pipelines software is required to be portable across many compute platforms ranging from high-performance computing centers to laptops (refer to DMS-REQ-0308 in :lse:`61`).
```

Reply:
I'm going to write this as “the Science Pipelines are...”, but otherwise :+1:.